core/domain/voiceover_services.py

```diff
@@ -129,19 +129,20 @@ def get_all_language_accent_codes_for_voiceovers(
     return language_codes_mapping
 
 
-def create_entity_voiceovers_model_instance(
-    entity_voiceovers_instance: voiceover_domain.EntityVoiceovers
+def create_entity_voiceovers_model(
+    entity_voiceovers: voiceover_domain.EntityVoiceovers
 ) -> voiceover_models.EntityVoiceoversModel:
-    """Creates an entity voiceovers model instance.
+    """Creates an entity voiceovers model instance without putting it into
```

Comment:
Creates --> Creates and returns

Put a comma after "instance"

Reply:
Done